index.html

```diff
@@ -103,20 +118,20 @@ <h2 class="heading-underline">Upcoming events</h2>
 						</article>
 						<article class="article row">
 							<a href="#">
-								<img class="img-fluid" src="https://raw.githubusercontent.com/Code-Your-Future/syllabus/master/html-css/assets/bikes-for-refugees-logo.jpg">
+								<img class="img-fluid" src="images/bikes-for-refugees_logo.jpg" style=" width:50px; height:50px;">
 							</a>
 							<div class="article-title article-title--sidebar">
 								<a href="#">Bike dropoff event in Edinburgh on May 11</a>
 							</div>
 						</article>
 						<article class="article row">
```

Comment:
Watch out, you cannot have a `row` without `col-...` classes inside (on the children elements)